client/app/hosting/local-seo/hosting-local-seo.popover.html

```diff
@@ -0,0 +1,10 @@
+<div class="popover-menu">
+    <button class="btn btn-link" type="button"
+            data-i18n-static="hosting_tab_LOCAL_SEO_access_interface"
+            data-ng-click="ctrl.goToInterface(location)">
+    </button>
+    <button class="btn btn-link" type="button"
+            data-i18n-static="hosting_tab_LOCAL_SEO_delete"
+            data-ng-click="setAction('local-seo/delete/hosting-local-seo-delete', location)">
+    </button>
+</div>
```

Comment:
Pay attention to the `.editorconfig` rules:) (Also refer to my previous comment about `<oui-datagrid>` component)